hera_sim/simulate.py

```diff
@@ -888,6 +888,9 @@ def func(freqs, ants, other=None):
         model_params = {k: v for k, v in model_params.items() if v is inspect._empty}
 
         # Pull the LST and frequency arrays if they are required.
+        # TODO: update this to allow more flexibility in which arguments are
+        # looked for (this is exactly what needs to be updated to accommodate the
+        # updated crosstalk model for IDR3 validation)
```

Comment:
pinning a TODO

Reply:
Ah, ok, the parenthetical isn't true (since I just kinda hardwired some stuff to make it work), but I still want to address the other bit. Do you think it's important enough to get this done in this PR, or should I punt this to another PR?

Reply:
let's postpone to another PR.